FileUpload/src/FileUpload.jsx

```diff
@@ -466,7 +472,7 @@ FileUpload.defaultProps = {
     linkInputActionText: "Press {save|[ENTER]} to save, or {cancel|[ESC]} to cancel",
     uploadCompleteText: "Upload Complete",
     uploadingText: "Uploading...",
-    uploadDefaultText: "myImage.jpg",
+    uploadDefaultText: "",
```

Comment:
And remove this.

Reply:
We still use this property. Only default values have been changed.

Reply:
Ok, I see it actually being used in Persona Bar components. Wonder if it's also `"myImage.jpg"` :joy: